app/src/main/java/fr/neamar/kiss/result/AppResult.java

```diff
@@ -161,7 +162,7 @@ private void launchEditTagsDialog(final Context context, final AppPojo app) {
         builder.setTitle(context.getResources().getString(R.string.tags_add_title));
 
         // Create the tag dialog
-        final View v = LayoutInflater.from(context).inflate(R.layout.tags_dialog, null);
+        @SuppressLint("InflateParams") final View v = LayoutInflater.from(context).inflate(R.layout.tags_dialog, null);
```

Comment:
same as before. try with **View.inflate** so we don't have `import android.annotation.SuppressLint;` and `@SuppressLint` everywhere